engine/tutorials/dockervolumes.md

```diff
@@ -151,6 +151,17 @@ $ docker run -d -P --name web -v /src/webapp:/webapp:ro training/webapp python a
 Here you've mounted the same `/src/webapp` directory but you've added the `ro`
 option to specify that the mount should be read-only.
 
+You can also relax the consistency requirements of a mounted directory
+to improve performance by adding the `cached` option:
+
+```bash
+$ docker run -d -P --name web -v /src/webapp:/webapp:cached training/webapp python app.py
+```
+
+The `cached` option typically improves the performance of read-heavy workloads on
+Docker for Mac, at the cost of some temporary inconsistency between the host and
+the container.  On other platforms `cached` currently has no effect.
```

Comment:
Link to the blog post from here?  A lot of people will probably run across this option from the tutorial and want to know more.